src/gluonnlp/models/electra.py

```diff
@@ -337,6 +347,58 @@ def get_initial_embedding(self, F, inputs, token_types=None):
         embedding = self.embed_dropout(embedding)
         return embedding
 
+    def apply_layerwise_decay(self, layerwise_decay, not_included=[]):
```

Comment:
Usually, we need to avoid setting default value to list. Instead, we may use a None.